app/helpers/react_components/mentoring/discussion.rb

```diff
@@ -63,6 +63,7 @@ def iterations
             unread: ccs.reject { |(_, seen), _| seen }.present?,
             created_at: iteration.created_at.iso8601,
             tests_status: iteration.tests_status,
+            automated_feedback: iteration.automated_feedback,
```

Comment:
This will lead to some painful n+1s so I'll need to pre-calculate that. 

```suggestion
            # TODO: Precalculate this to avoid n+1s
            automated_feedback: iteration.automated_feedback,
```